src/Source/PointCloudSource.js

```diff
@@ -0,0 +1,67 @@
+import Source from 'Source/Source';
+import Fetcher from 'Provider/Fetcher';
+import PotreeBinParser from 'Parser/PotreeBinParser';
+import PotreeCinParser from 'Parser/PotreeCinParser';
+
+/**
+ * @classdesc
+ * PointCloudSource are object containing informations on how to fetch points cloud resources.
+ *
+ */
+class PointCloudSource extends Source {
+    /**
+     * @param {Object} source - An object that can contain all properties of a
+     * PointCloudSource
+     * @param {string} source.url - folder url.
+     * @param {string} source.file - file name.
+     *
+     * @extends Source
+     *
+     * @constructor
+     */
+    constructor(source) {
+        if (!source.url) {
+            throw new Error('New PointCloudSource: url is required');
+        }
+        if (!source.file) {
+            throw new Error('New PointCloudSource: file is required');
+        }
+
+        super(source);
+        this.file = source.file;
+        this.url = source.url;
+        this.fetcher = Fetcher.arrayBuffer;
+        this.extensionOctree = 'hrc';
+
+        // For cloud specification visit:
+        // https://github.com/PropellerAero/potree-propeller-private/blob/master/docs/file_format.md#cloudjs
```

Comment:
Please add a quick copy of this in the documentation of the `constructor` above, in case this page disappears, and to ease the process to the user

Reply:
done